python/tests/expressions/test_expressions.py

```diff
@@ -56,7 +56,14 @@
     Or,
     Reference,
 )
-from pyiceberg.expressions.literals import literal
+from pyiceberg.expressions.literals import (
+    FloatAboveMax,
+    FloatBelowMin,
+    IntAboveMax,
+    IntBelowMin,
```

Comment:
Should we do the same thing for `Long`? I think Python's `int` can represent numbers that are larger than an 8-byte signed integer.

Reply:
Yes, this is possible:
```python
➜  python git:(fd-above-max-below-min) python3
Python 3.10.8 (main, Oct 13 2022, 09:48:40) [Clang 14.0.0 (clang-1400.0.29.102)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from pyiceberg.types import LongType
>>> LongType.max + 1
9223372036854775808
>>> LongType.max * 2
18446744073709551614
```

Added 👍🏻

Reply:
I've implemented these also in the `to()` method. Using the `__new__` this might be exposed to the user because `literal(LongType.max + 1)` would return a `LongAboveMax` right away. Instead, we now check this at binding.